### PR TITLE
Add Event Hook plugins to DescribePlugins function

### PR DESCRIPTION
**Pull request by hacdias**

### CODE REVIEW

File: plugins.go [hunk 1]
```diff
@@ -65,6 +65,11 @@ func DescribePlugins() string {
 			others = append(others, s)
 		}
 	}
+
+	for hookPlugin := range eventHooks {
+		others = append(others, hookPlugin)
+	}
+
```
Comment (mholt) [inline plugins.go]: Move this above the "Let's alphabetize these" comment and give it their own section: "Event hooks" - you can just print the name as you iterate them. Thanks!

Comment (hacdias) [inline plugins.go]: Should they be prefixed with "hook."?

Comment (mholt) [inline plugins.go]: Yes. :) That would be good.

**Comment by hacdias** [comment]
Done!

**Comment by mholt** [comment]
Spoke too soon. I didn't notice the tests are failing. How about only printing event hooks (including the header) if there are at least one; make sure the test passes.

**Pull request merged**